### ISSUE TITLE
prune mode does not work

### ISSUE THREAD

**Issue by kr1z1s** [superbitcoin/SuperBitcoin#82]
OS Linux

Daemon compile from source

Prune mode does not work

In sbtc.conf - prune=700

<img width="1126" alt="Скриншот 2019-12-27 12 21 46" src="https://user-images.githubusercontent.com/5347595/71511454-796a2080-28a3-11ea-8c50-153c4848fe97.png">
In logs

But blockchain not prunning

**Comment by laakie** [comment]
Indeed, same here, Very concerning as we need to carry the whole BTC history....
Very inefficient to not be pruning it

**Comment by crackfoo** [comment]
Can someone from dev address this please?

Thanks
